consul_kv.go

```diff
@@ -0,0 +1,52 @@
+package onlinelab
+
+import (
+	"github.com/hashicorp/consul/api"
+	"github.com/json-iterator/go"
+)
+
+var json = jsoniter.ConfigCompatibleWithStandardLibrary
+var kv *api.KV
```

Comment:
Put these vars into ConsulConfigStorage struct as a field.
Please try to avoid to use global vars.